src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -35,8 +35,7 @@ const TableHeader: React.FC<TableHeaderProps> = ({
       (user: User) => user.data?.[venueName]?.table === seatedAtTable
     );
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const firestoreUpdate = (doc: string, update: any) => {
+  const firestoreUpdate = (doc: string, update: firestore.UpdateData) => {
```

Comment:
This change is causing a TypeError currently

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/793/commits/ab1f717fcfffe2690e1eb95d241d509e6d119236